From f6cecd95af59748a4a05968d2f0dad7d1e5efb7b Mon Sep 17 00:00:00 2001
From: Tung Huynh <tung75605@gmail.com>
Date: Sun, 10 Dec 2023 22:29:16 -0800
Subject: [PATCH] ffmpeg: dont error on int conversion

---
 contrib/src/ffmpeg/rules.mak | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/src/ffmpeg/rules.mak b/contrib/src/ffmpeg/rules.mak
index 9baf9b140d..121721a9bc 100644
--- a/contrib/src/ffmpeg/rules.mak
+++ b/contrib/src/ffmpeg/rules.mak
@@ -262,7 +262,7 @@ endif
 	$(MOVE)
 
 .ffmpeg: ffmpeg
-	cd $< && $(HOSTVARS) ./configure \
+	cd $< && $(HOSTVARS) CFLAGS="$(CFLAGS) -Wno-error=int-conversion" ./configure \
 		--extra-ldflags="$(LDFLAGS)" $(FFMPEGCONF) \
 		--prefix="$(PREFIX)" --enable-static --disable-shared
 	cd $< && $(MAKE) install-libs install-headers
-- 
2.42.0.windows.2

