From 2dfa38e1615c4a96b1470b1b07fb2819a5b20cf6 Mon Sep 17 00:00:00 2001
From: Martin Finkel <me@martinfinkel.com>
Date: Tue, 28 Jan 2025 12:54:57 +0700
Subject: [PATCH 1/1] invoke appcontainer.pl

---
 extras/package/win32/build.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/extras/package/win32/build.sh b/extras/package/win32/build.sh
index bf709458aa..b724c45c4d 100755
--- a/extras/package/win32/build.sh
+++ b/extras/package/win32/build.sh
@@ -523,6 +523,9 @@ elif [ "$INSTALLER" = "r" ]; then
 make package-win32
 elif [ "$INSTALLER" = "u" ]; then
 make package-win32-release
+if [ ! -z "$WINSTORE" ]; then
+    find . \( -name "*.dll" -o -name "*.exe" \) -exec ../../patches/appcontainer.pl {} \;
+fi
 sha512sum vlc-*-release.7z
 elif [ -n "$INSTALL_PATH" ]; then
 make package-win-install
-- 
2.25.1

