From b1f601ad759389e9cf910c987b7c7ef638b8232f Mon Sep 17 00:00:00 2001
From: Tung Huynh <tung75605@gmail.com>
Date: Sun, 10 Dec 2023 23:04:49 -0800
Subject: [PATCH 2/7] gcrypt: dont error on incompatible function pointer types

---
 contrib/src/gcrypt/rules.mak | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/contrib/src/gcrypt/rules.mak b/contrib/src/gcrypt/rules.mak
index 7319fc046c..7dbd0bc022 100644
--- a/contrib/src/gcrypt/rules.mak
+++ b/contrib/src/gcrypt/rules.mak
@@ -39,15 +39,15 @@ GCRYPT_CONF = \
 	--enable-pubkey-ciphers=dsa,rsa,ecc \
 	--disable-docs
 
+GCRYPT_EXTRA_CFLAGS = -Wno-error=incompatible-function-pointer-types
+
 ifdef HAVE_WIN32
 ifeq ($(ARCH),x86_64)
 GCRYPT_CONF += --disable-asm --disable-padlock-support
 endif
 endif
 ifdef HAVE_IOS
-GCRYPT_EXTRA_CFLAGS = -fheinous-gnu-extensions
-else
-GCRYPT_EXTRA_CFLAGS =
+GCRYPT_EXTRA_CFLAGS += -fheinous-gnu-extensions
 endif
 ifdef HAVE_MACOSX
 GCRYPT_CONF += --disable-aesni-support
-- 
2.42.0.windows.2

