From 5291be0bdf706fc63ba291df1271ee99326f61bc Mon Sep 17 00:00:00 2001
From: Jean-Baptiste Kempf <jb@videolan.org>
Date: Wed, 16 Apr 2014 16:09:53 +0200
Subject: [PATCH 02/30] Cheat for Windows Store subtitles

---
 lib/video.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/lib/video.c b/lib/video.c
index acbba3a30f..f4e9e2fba3 100644
--- a/lib/video.c
+++ b/lib/video.c
@@ -406,8 +406,13 @@ int libvlc_video_set_subtitle_file( libvlc_media_player_t *p_mi,
         char* psz_mrl = vlc_path2uri( psz_subtitle, NULL );
         if( psz_mrl )
         {
+#if VLC_WINSTORE_APP
+            if( !input_AddSlave( p_input_thread, SLAVE_TYPE_SPU, psz_mrl,
+                                 false, false, false ) )
+#else
             if( !input_AddSlave( p_input_thread, SLAVE_TYPE_SPU, psz_mrl,
                                  true, false, false ) )
+#endif
                 b_ret = true;
             free( psz_mrl );
         }
-- 
2.26.0.windows.1

