From e2d4a2504d850d54b3db9a6ab753f3cdaaf4055f Mon Sep 17 00:00:00 2001
From: Tung Huynh <tung75605@gmail.com>
Date: Mon, 11 Dec 2023 03:04:04 -0800
Subject: [PATCH 3/7] gnutls: fix clang errors

---
 contrib/src/gnutls/rules.mak | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/contrib/src/gnutls/rules.mak b/contrib/src/gnutls/rules.mak
index c6eeeccbd4..2ac81dc5e1 100644
--- a/contrib/src/gnutls/rules.mak
+++ b/contrib/src/gnutls/rules.mak
@@ -70,6 +70,7 @@ GNUTLS_CONF := \
 	$(HOSTCONF)
 
 GNUTLS_ENV := $(HOSTVARS)
+GNUTLS_EXTRA_CFLAGS = -Wno-error=implicit-function-declaration -Wno-error=int-conversion
 
 DEPS_gnutls = nettle $(DEPS_nettle)
 
@@ -90,7 +91,7 @@ endif
 endif
 
 .gnutls: gnutls
-	cd $< && $(GNUTLS_ENV) ./configure $(GNUTLS_CONF)
+	cd $< && $(GNUTLS_ENV) CFLAGS="$(CFLAGS) $(GNUTLS_EXTRA_CFLAGS)" ./configure $(GNUTLS_CONF)
 	cd $< && $(MAKE) -C gl install
 	cd $< && $(MAKE) -C lib install
 	touch $@
-- 
2.42.0.windows.2

