From 87bdcd823e6b52dead3b912c5283ec7bb7e85ac4 Mon Sep 17 00:00:00 2001
From: Jean-Baptiste Kempf <jb@videolan.org>
Date: Wed, 12 Mar 2014 00:11:26 +0100
Subject: [PATCH 03/30] Accept the weird WinRT pathes

---
 src/text/url.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/text/url.c b/src/text/url.c
index c3c36015d7..c9af2f8838 100644
--- a/src/text/url.c
+++ b/src/text/url.c
@@ -176,6 +176,11 @@ char *vlc_path2uri (const char *path, const char *scheme)
         }
     }
     else
+#if VLC_WINSTORE_APP
+    if ((!strncmp(path, "file://{", 8) || !strncmp(path, "winrt://{", 9) || path[0] == '{') && strlen(path) > 32)
+        return strdup( path );
+    else
+#endif
     if (!strncmp (path, "\\\\", 2))
     {   /* Windows UNC paths */
         /* \\host\share\path -> file://host/share/path */
-- 
2.26.0.windows.1

