From 0451bb7dce68f5f5b95b7f36516bf303b633c744 Mon Sep 17 00:00:00 2001
From: Tung Huynh <tung75605@gmail.com>
Date: Mon, 11 Dec 2023 14:06:58 -0800
Subject: [PATCH 5/5] contrib: apply clang fix for all contribs

---
 contrib/src/main.mak | 1 +
 1 file changed, 1 insertion(+)

diff --git a/contrib/src/main.mak b/contrib/src/main.mak
index 2a7776caa1..00aa6a83a4 100644
--- a/contrib/src/main.mak
+++ b/contrib/src/main.mak
@@ -165,6 +165,7 @@ endif
 
 ifneq ($(findstring clang, $(shell $(CC) --version)),)
 HAVE_CLANG := 1
+EXTRA_CFLAGS += -Wno-error=implicit-function-declaration -Wno-error=incompatible-function-pointer-types -Wno-error=int-conversion
 endif
 
 cppcheck = $(shell $(CC) $(CFLAGS) -E -dM - < /dev/null | grep -E $(1))
-- 
2.42.0.windows.2

